plume-front/src/main.rs

```diff
@@ -46,10 +46,15 @@ lazy_static! {
         let catalogs = include_i18n!();
         let lang = js! { return navigator.language }.into_string().unwrap();
         let lang = lang.splitn(2, '-').next().unwrap_or("en");
+
+        let english_position = catalogs
```

Comment:
this should be either a (cached?) constant, or a (cached?) config value

Reply:
It should be a constant yes, but the rust doc of constant says : 

> Constants can be set only to a constant expression and not to the result of a function call or any other value that will be computed at runtime.
So i thought the keyword const was forbidden in this case.

Reply:
All this is in a lazy_static! call, so the result will be evaluated only one time, the first time CATALOG is accessed. The only thing it's not is cached between two different page loading, but that's not necessary imo